Automatic Validation ended with:
> 2025-02-03T12:30:05.7048394Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2.8.2, 2.8.2]]
 2025-02-03T12:30:05.7060077Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2.8.2, 2.8.2]]

(Automated response - build 965.)

--- a/manifests/t/TeamLidarr/Lidarr/2.9.6.4552/TeamLidarr.Lidarr.installer.yaml
+++ b/manifests/t/TeamLidarr/Lidarr/2.9.6.4552/TeamLidarr.Lidarr.installer.yaml
@@ -16,9 +16,9 @@ InstallerSwitches:
   Custom: /NORESTART
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
-- DisplayName: Lidarr version 2.8.2
+- DisplayName: Lidarr version 2.9.6
   Publisher: Team Lidarr
-  DisplayVersion: 2.8.2
+  DisplayVersion: 2.9.6
 Installers:
 - Architecture: x86
   InstallerUrl: https://github.com/Lidarr/Lidarr/releases/download/v2.9.6.4552/Lidarr.master.2.9.6.4552.windows-core-x86-installer.exe

